standard_test.go

```diff
@@ -763,71 +763,90 @@ func TestSnakeHasLostHeadToHead(t *testing.T) {
 
 func TestEliminateSnakes(t *testing.T) {
 	tests := []struct {
+		Name                     string
```

Comment:
I added the name to all the tests, this allows us to isolate a single test in the list of tests, which makes debugging a single test infinitely easier.

![image](https://user-images.githubusercontent.com/326884/74487110-8a7ee900-4e73-11ea-88ae-262774f38bf6.png)